src/sceneRoute.tsx

```diff
@@ -102,6 +104,17 @@ const Routes = (props: { routeCategories: RouteCategories; header: string }) =>
   const categoriesShown = useHookstate({} as Record<string, boolean>)
   const hidden = useHookstate(false)
 
+  useImmediateEffect(() => {
+    createEngine()
```

Comment:
we shouldnt need to create and destroy the engine. the engine can be created in module scope of the root file imported as a route